Cloud Pak for Data/WML/notebooks/misc/Watson OpenScale and Watson ML Engine with AI-function.ipynb

```diff
@@ -161,7 +162,7 @@
     "                   \"username\": WOS_CREDENTIALS['username'],\n",
     "                   \"password\": WOS_CREDENTIALS['password'],\n",
     "                   \"instance_id\": \"wml_local\",\n",
-    "                   \"version\" : \"3.5\" #If your env is CP4D 4.0 then specify \"4.0\" instead of \"3.5\"  \n",
+    "                   \"version\" : \"4.5\" #If your env is CP4D 4.0 then specify \"4.0\" instead of \"3.5\"  \n",
```

Comment:
This comment needs to change, since we are specifying 4.5 instead of 3.5 by default